packages/nectar/src/sagas/socket/startConnection/startConnection.saga.ts

```diff
@@ -45,7 +44,16 @@ export function subscribe(socket: Socket) {
   | ReturnType<typeof identityActions.throwIdentityError>
   | ReturnType<typeof communitiesActions.storePeerList>
   | ReturnType<typeof communitiesActions.updateCommunity>
-  >(emit => {
+  | ReturnType<typeof connectionActions.addInitializedCommunity>
+  | ReturnType<typeof connectionActions.addInitializedRegistrar>
+  | ReturnType<typeof connectionActions.addConnectedPeers>
+  >((emit) => {
+    socket.on(SocketActionTypes.PEER_CONNECT, (payload: { connectedPeers: ConnectedPeers }) => {
+      emit(connectionActions.addConnectedPeers(payload.connectedPeers))
+    })
+    socket.on(SocketActionTypes.PEER_DISCONNECT, (payload: { connectedPeers: ConnectedPeers }) => {
```

Comment:
Unnecessarily duplicated event. That can be done using just one type (e.g. connectedPeers)